introduction.md

```diff
@@ -1,4 +1,8 @@
 Hi, I am a new dojo. Greetings!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?